packages/webpack/src/config/server.js

```diff
@@ -123,7 +125,7 @@ export default class WebpackServerConfig extends WebpackBaseConfig {
         maxEntrypointSize: Infinity,
         maxAssetSize: Infinity
       },
-      externals: [].concat(config.externals || [])
+      externals: ['node-fetch-native'].concat(config.externals || [])
```

Comment:
Is it still essential with mjs fix?

Reply:
This is still required, yes, as it is not called with the full path here but just the specifier (`node-fetch-native`).